src/components/TableView/index.js

```diff
@@ -0,0 +1,66 @@
+import {
+  Box,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@mui/material";
+import React from "react";
+import CopyIcon from "../../app/assets/svg/CopyIcon.svg";
+import ChevronDown from "../../app/assets/svg/ChevronDown.svg";
+import Image from "next/image";
+
+const TableView = () => {
```

Comment:
Same, this component can make use of that common component we can make using a library like react table. parent will pass the data.
Also name "TableView" can be more self explanatory